docs/data/material/components/grid2/grid2.md

```diff
@@ -32,7 +32,7 @@ From now on, the `Grid` v1 and `Grid` v2 refer to the import as:
 
 ```js
 import Grid from '@mui/material/Grid'; // Grid version 1
-import Grid2 from '@mui/material/Unstable_Grid2'; // Grid version 2
+import Grid from '@mui/material/Unstable_Grid2'; // Grid version 2
```

Comment:
For consistency with the text above. Reported in https://mui-org.slack.com/archives/C041SDSF32L/p1681137941293499.